code/FulpstationCode/fulp_starter_gear/fulp_starter_gear.dm

```diff
@@ -0,0 +1,14 @@
+/obj/item/storage/belt/security/fulp_starter_full/PopulateContents()
+	new /obj/item/reagent_containers/spray/pepper(src)
+	new /obj/item/flashlight/seclite(src)
+	new /obj/item/radio(src)
+	new /obj/item/assembly/flash/handheld(src)
+	new /obj/item/melee/baton/loaded(src)
+	update_icon()
+
+/obj/item/storage/box/security/improved/PopulateContents()
+	..() // we want the regular stuff too

```

Comment:
On a complete side note, using ..() in objects to inherit inventories is a really good idea, I'll absolutely be trying this in #168 to see if I can make the vendor overwrites smaller.